Yoast/wordpress-seo
# Only render content when post is not singular

## Summary

This PR can be summarized in the following changelog entry:

* Fixes a bug where structured data was rendered on pages with multiple posts.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Repeat the steps in #10965, and make sure no json/ld is rendered.

## Quality assurance


Fixes #10965
